### ISSUE TITLE
Url feature

### ISSUE THREAD

**Issue by Warden20** [mmagnus/PocketX-Calibre-Plugin#3]
It would be awesome if they could implement the feature of adding the url below the title to the main recipe.

**Comment by mmagnus** [comment]
Yeah, I know, but the problem is that this is done in some hacky way, and still work like in 90%, pretty good but I have to find a way to fix it to 100% :(
